backend/spec/requests/books_spec.rb

```diff
@@ -33,10 +40,18 @@
   end
 
   describe "GET /api/v1/books" do
-    subject(:request) { get("/api/v1/books/", params: params) }
+    subject(:request) { get("/api/v1/books/", params: params, headers: headers) }
+
+    let(:params) { { page: 5 } }
 
     before { request }
 
+    context "when requesting client is unauthorized" do
+      let(:headers) { {} }
+
+      it { expect(error_messages).not_to(be_nil) }
+    end
```

Comment:
I see duplciated test. You can make use of rspec's shared_examples to dry this up - https://www.betterspecs.org/#shared